impl/client/src/main/javascript/web/vizapi/ccc/sample/sampleVisualization.js

```diff
@@ -123,7 +124,7 @@ require([
 
     }
 
-    this.numSpan.innerText = value == null ? "" : value;
+    xssUtil.setHtml(this.numSpan, value == null ? "" : value);
```

Comment:
`innerText` was fine, here.

Reply:
updated

Reply:
No changes are needed on this file. You forgot about the dependency and argument.

Reply:
Updated